src/improvement-1.js

```diff
@@ -1,10 +1,3 @@
-/**
- *  TRY TO IMPROVE THE FOLLOWING FUNCTION.
- *  YOU CAN REPLACE THE CODE INSIDE THE FUNCTIONS, THE FUNCTION NAME, ETC.
- *  PLEASE DO NOT REMOVE `export default function`
- */
-
-// FUNCTION 1
  const countUpperCaseRs = letters => letters.match(/R/g).length;
```

Comment:
Leave comments: can you consider to leave some comment to explain what this regex do?